workspace-manager-clienttests/src/main/java/scripts/disruptivescripts/DeleteInitialPods.java

```diff
@@ -0,0 +1,69 @@
+package scripts.disruptivescripts;
+
+import bio.terra.testrunner.common.utils.KubernetesClientUtils;
+import bio.terra.testrunner.runner.DisruptiveScript;
+import bio.terra.testrunner.runner.config.TestUserSpecification;
+import io.kubernetes.client.openapi.models.V1Deployment;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+import java.util.stream.Collectors;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class DeleteInitialPods extends DisruptiveScript {
+  private static final Logger logger = LoggerFactory.getLogger(DeleteInitialPods.class);
+
+  public DeleteInitialPods() {
+    super();
+    manipulatesKubernetes = true;
+  }
+
+  protected static final int SECONDS_TO_WAIT_BEFORE_STARTING_DISRUPT = 15;
+
+  @Override
+  public void disrupt(List<TestUserSpecification> testUsers) throws Exception {
+    // give user journey threads time to get started before disruption
+    TimeUnit.SECONDS.sleep(SECONDS_TO_WAIT_BEFORE_STARTING_DISRUPT);
+
+    logger.info(
+        "Starting disruption - all initially created api pods will be deleted, one by one.");
+
+    String componentLabelKey =
+        KubernetesClientUtils.getComponentLabel(); // e.g. "app.kubernetes.io/component";
+    String componentLabelVal =
+        KubernetesClientUtils.getApiComponentLabel(); // e.g. "workspacemanager";
+
+    V1Deployment workspacemanagerDeployment = KubernetesClientUtils.getApiDeployment();
+    if (workspacemanagerDeployment == null) {
+      throw new RuntimeException("WorkspaceManager deployment not found.");
+    }
+
+    List<String> podsToDelete =
+        KubernetesClientUtils.listPods().stream()
+            .filter(
+                pod ->
+                    pod.getMetadata().getLabels().containsKey(componentLabelKey)
+                        && pod.getMetadata()
+                            .getLabels()
+                            .get(componentLabelKey)
+                            .equals(componentLabelVal))
+            .map(pod -> pod.getMetadata().getName())
+            .collect(Collectors.toList());
```

Comment:
nit: Could you add a TODO here with the ticket number to refactor this out into a method of the Test Runner KubernetesClientUtils class?

Reply:
Here's the ticket: [https://broadworkbench.atlassian.net/browse/QA-1421](https://broadworkbench.atlassian.net/browse/QA-1421)

Reply:
ok thanks for the ticket. I meant to add it as a comment in the code (e.g. "// TODO (QA-1421): refactor this out into a method of the Test Runner KubernetesClientUtils class")

Reply:
Added TODO comment.